modules/simple_forms_api/app/models/simple_forms_api/vha_10_10d.rb

```diff
@@ -25,12 +25,8 @@ def metadata
     def handle_attachments(file_path)
       attachments = get_attachments
       if attachments.count.positive?
-        combined_pdf = CombinePDF.new
-        combined_pdf << CombinePDF.load(file_path)
         attachments.each do |attachment|
```

Comment:
Do we need this loop? looks like there's no action inside it

Reply:
Yes we do. The attachments is an array that stores the attachments from the forms.